src/shelly-device.ts

```diff
@@ -59,11 +59,12 @@ export class ShellyDevice extends Device {
     }
 
     private configureRollerPosition(): void {
-        new RollerPositionProperty(this, 'rollerPosition', 'Relative roller shutter position', (value) => {
+        const rollerPositionProperty = new RollerPositionProperty(this, 'rollerPosition', 'Opening level in %', (value) => {
             this.device.setRollerPosition(value).catch(reason => {
                 console.log(reason);
             })
         });
+        rollerPositionProperty.setValue(this.device.rollerPosition);
```

Comment:
I think this needs to be `setCachedValueAndNotify`.
The method `setValue` will also send the value to the shelly.
```suggestion
        rollerPositionProperty.setCachedValueAndNotify(this.device.rollerPosition);
```